src/Artsy/Analytics/Schema/Values.ts

```diff
@@ -36,263 +36,136 @@ export enum OwnerType {
+  SearchedAutosuggestWithResults = "Searched from header with results",
   SelectedItemFromSearch = "Selected item from search",
   SelectedItemFromSearchPage = "Selected item from search page",
-  SearchedAutosuggestWithResults = "Searched from header with results",
-  SearchedAutosuggestWithoutResults = "Searched from header with no results",
+  SubmittedCounterOffer = "submitted_counter_offer",
+  SubmittedOffer = "submitted_offer",
+  SubmittedOrder = "submitted_order",
+  Tap = "Tap", // TODO: Check if ‘Click’ and this can be combined.
+  ViewedOfferHigherThanListPrice = "Viewed offer higher than listed price",
+  ViewedOfferTooLow = "Viewed offer too low",
+  ViewedProduct = "Product Viewed",
 }
 
 /**
  * The identifier that ties an interaction to a result.
  */
 export enum ActionName {
-  /**
-   * Artist Page
-   */
   ArtistFollow = "artistFollow",
   ArtistUnfollow = "artistUnfollow",
   ArtworkAboutTheWork = "Artwork about the work",
-
-  /**
-   * Authentication
-   */
-  ViewEditorial = "viewed editorial",
+  BidFlowAddBillingAddress = "addBillingAddress",
+  BidFlowPlaceBid = "placeBid",
+  BidFlowSaveBillingAddress = "saveBillingAddress",
+  ConsignmentDraftCreated = "consignmentDraftCreated",
+  ConsignmentInterest = "Interested in selling a work learn more", // TODO: Old schema
+  ConsignmentSubmitted = "consignmentSubmitted",
+  ConversationLink = "conversationLinkUsed",
+  ConversationSendReply = "conversationSendReply",
   Dismiss = "dismiss",
   EmailNextButton = "emailNextButton",
-  PasswordNextButton = "passwordNextButton",
-
-  /**
-   * Gene Page
-   */
   GeneFollow = "geneFollow",
   GeneUnfollow = "geneUnfollow",
-
-  /**
-   * Home page events
-   */
-  HomeArtistRailFollow = "homeArtistRailFollow",
   HomeArtistArtworksBlockFollow = "homeArtistArtworksBlockFollow",
-
-  /**
-   * Conversations / Inbox / Messaging
-   */
-  ConversationSendReply = "conversationSendReply",
-  ConversationLink = "conversationLinkUsed",
+  HomeArtistRailFollow = "homeArtistRailFollow",
   InquiryCancel = "inquiryCancel",
   InquirySend = "inquirySend",
-
-  /**
-   *  Saves And Follows Events
-   */
-  SavesAndFollowsWorks = "savesAndFollowsWorks",
-  SavesAndFollowsArtists = "savesAndFollowsArtists",
-  SavesAndFollowsCategories = "savesAndFollowsCategories",
-
-  /**
-   *  Consignment flow
-   */
-  ConsignmentDraftCreated = "consignmentDraftCreated",
-  ConsignmentSubmitted = "consignmentSubmitted",
-  ConsignmentInterest = "Interested in selling a work learn more", // TODO: Old schema
-
-  /**
-   * Bid flow
-   */
-  BidFlowAddBillingAddress = "addBillingAddress",
-  BidFlowPlaceBid = "placeBid",
-  BidFlowSaveBillingAddress = "saveBillingAddress",
-
-  /**
-   * Generic
-   */
-  ReadMoreExpanded = "readMoreExpanded", // TODO: This differs from old event
   InSale = "In current auction", // TODO: Old schema
   InShow = "In featured show", // TODO: Old schema
+  PasswordNextButton = "passwordNextButton",
```

Comment:
(Another example of what I mentioned above, this seems more like a ‘subject’.)

Reply:
Not sure what this relates to -- this change is just from alpha; not part of the tracking matrix that data-team passed over. (See latest diff for Header-only related changes.)

Reply:
Yeah, no, I know you didn’t add this. I meant this in relation to my comment that said:

> some of the newer ones don’t (e.g. `SubmittedOffer`, which was probably a click with the action `submitted_offer`).